modules/ROOT/nav.adoc

```diff
@@ -289,7 +290,6 @@
 *** PowerPaste
 **** xref:introduction-to-powerpaste.adoc[Introduction]
 **** xref:powerpaste-options.adoc[Options]
-**** xref:powerpaste-toolbars-menus.adoc[Toolbar buttons and menu items]
```

Comment:
Just checking this was intentional?

Reply:
Yes, all the UI components are in core now.